RecoHI/HiCentralityAlgos/python/HiCentrality_cfi.py

```diff
@@ -36,11 +36,11 @@
 
 from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
-for e in [pp_on_XeXe_2017, pp_on_AA_2018]:
-    e.toModify(hiCentrality,
-               producePixelTracks = True,
-               srcPixelTracks = "hiConformalPixelTracks",
-               srcTracks = cms.InputTag("generalTracks"),
-               srcVertex = cms.InputTag("offlinePrimaryVertices")
-               )
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+(pp_on_XeXe_2017 | pp_on_AA_2018 | pp_on_PbPb_run3).toModify(hiCentrality,
+                                                             producePixelTracks = True,
+                                                             srcPixelTracks = "hiConformalPixelTracks",
+                                                             srcTracks = cms.InputTag("generalTracks"),
```

Comment:
Remove type specification for the modified `srcTracks` and `srcVertex`:
```PYTHON
                                                             srcTracks = "generalTracks",
                                                             srcVertex = "offlinePrimaryVertices"
```

Reply:
Fixed in the latest commit, sorry I missed that. 
Looks like I have another crash to fix that arrived in parallel though.